docs/content/en/contribute/docs/source-file-structure/_index.md

```diff
@@ -54,6 +54,118 @@ In the order they appear in the rendered docs, the subdirectories are:
   Information to help users who are migrating to Keptn
   from Keptn v1
 
+### Working with reference pages
+
+The Keptn documentation includes two reference sections
+that document the Keptn APIs and CRDs.
+For background information, see:
+
+* [Kubernetes API Concepts](https://kubernetes.io/docs/reference/using-api/api-concepts/)
+* [Kubernetes API Reference](https://kubernetes.io/docs/reference/kubernetes-api/)
+
+#### API Reference
+
+The
+[API Reference](https://lifecycle.keptn.sh/docs/crd-ref/)
+pages are autogenerated from the source code.
+This is a comprehensive list of all APIs and resources Keptn uses,
+most of which users seldom see.
+
+Descriptive text for the APIs is authored in the source code itself.
+Each operator has its own API library with different versions.
+The source locations are:
+
+* [Lifecycle API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/lifecycle)
+* [Metrics API](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/api)
+* [Options API](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/apis/options/v1alpha1)
+
+The text is coded in a limited form of markdown.
+
+After modifying text for the API reference, you must regenerate it.
+
+To manuall install the tool to regenerate the API reference docs,
+execute the following command
+in the root directory of your `lifecycle-toolkit` clone:
+
+```shell
+go install github.com/elastic/crd-ref-docs@master
+
+```
```

Comment:
The script automatically installs the correct version of the `crd-ref-docs` tool. 
I would not suggest users install it on their own using the latest version since it could generate docs that are out-of-sync from the current ones.

Reply:
So what is the right thing to do?  Periodically, the CI tools give me an error with instructions to run this command (I think this is the command).even though my PR does not modify the api source files.  So are you saying that the instructions I have to install the command are obsolete and should not be included here?  But also that I should not be rebuilding these pages unless the system tells me to.  If I modify text for one of these APIs, is there a way to render the docs so I can check formatting?  Will netlify display the modified text if I don't generate

Reply:
I would also agree on removing the install part

Reply:
The CI just give you the command that @thisthat posted above, not the `go install` one. And @thisthat is correct that the installation is included in the script already.

Reply:
done